shared/naturalcrit/splitPane/splitPane.jsx

```diff
@@ -15,12 +15,13 @@ const SplitPane = createClass({
 
 	getInitialState : function() {
 		return {
-			currentDividerPos : null,
-			windowWidth       : 0,
-			isDragging        : false,
-			moveSource        : false,
-			moveBrew          : false,
-			showMoveArrows    : true
+			currentDividerPos  : null,
+			windowWidth        : 0,
+			isDragging         : false,
+			moveSource         : false,
+			moveBrew           : false,
+			viewablePageNumber : 0,
```

Comment:
`viewablePageNumber` doesn't seem to be used in `splitPane`

Reply:
I don't remember adding that one. is it maybe debris from merges?